operator/cmd/syncclusterconfig/sync.go

```diff
@@ -377,9 +392,18 @@ func (s *Syncer) Sync(ctx context.Context, desired map[string]any, usersTXT map[
 	if err != nil {
 		return nil, err
 	}
+	schema, err := s.Client.ClusterConfigSchema(ctx)
```

Comment:
Can we instead hoist this call to the top of this function so it only occurs once rather than repeating it?